build-manual.py

```diff
@@ -73,6 +73,78 @@ def __init__(self, number: str, title: str, filename: str):
 		self.filename = filename
 		self.items = []
 
+def make_one_page(dest_directory):
+	"""
+	Generate one-page php file of the manual.
+	"""
+	# Get all php files in destination directory.
+	php_files = list(filter(lambda x: x.endswith("php"), os.listdir(dest_directory)))
+	php_re = regex.compile(r"<\?.+?\?>", regex.S)
```

Comment:
Pre-compiling regexes isn't necessary with the `regex` library, can we get rid of this and the rest?